settings.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve().parent
+
+EXCEL_PATH = BASE_DIR.joinpath('data/my_operations.xls')
```

Comment:
синтаксически вернее так:
```suggestion
EXCEL_PATH = BASE_DIR.joinpath('data', 'my_operations.xls')
```